modules/ROOT/pages/wifi.adoc

```diff
@@ -0,0 +1,66 @@
+
+== How do I add the missing wifi-firmwares?
+
+Some wifi-firmwares will be split into sub-packages in Fedora 39 and Fedora 40.
+Fedora Cores OS will keep them in until Fedora 41, but display a
```

Comment:
The styling convention for asciidoc is one sentence per line.

This reduces the amount of diff while keeping things readable.

See: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line